### PR TITLE
more work on travis-ci testing

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
tox-travis: removed.

macOS: I took the easy approach here and just used the pythons I found already being installed on some xcode versions.

We do more effort for borgbackup, so if we want more for scandir also, all the scripting could be taken from there. But sometimes brew causes issues and building pythons via pyenv is rather slow, so not sure if we want that here also.

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
macOS: I took the easy approach here and just used the pythons I found already being installed on some xcode versions.

We do more effort for borgbackup, so if we want more for scandir also, all the scripting could be taken from there. But sometimes brew causes issues and building pythons via pyenv is rather slow, so not sure if we want that here also.

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +27,14 @@ matrix:
...
           dist: xenial
           env: TOXENV=py37
+        - language: generic
+          os: osx
+          osx_image: xcode9.4
+          env: TOXENV=py27,py36
+        - language: generic
+          os: osx
+          osx_image: xcode10.1
+          env: TOXENV=py27,py37
```
Comment (benhoyt) [inline .travis.yml]: Doesn't hurt, but we don't need to run macOS Python 2.7 here and also above. Maybe just `TOXENV=py37` here?

**Comment by ThomasWaldmann** [comment]
I noticed the duplication, but considering the very little effort to 
actually run the tests compared to the setup of the testing env, I 
thought I rather keep it.

Sometimes there are race conditions or other flaky behaviour, sometimes 
travis-ci breaks stuff (at borg, I am busy with restarting the tests all 
the time due to such stuff).

Also, the documentation says "there is no python", so we are lucky that 
this is actually not true. OTOH, some guy at travis might just deinstall 
some python some day...

**Comment by benhoyt** [comment]
Sounds good -- merged.

**Pull request merged**